README.md

```diff
@@ -1 +1,36 @@
-# capstone-1
\ No newline at end of file
+# Project Name
+
+- Conference page
+
+> Description the project.
+
+- A conference page for Industrial Biotechnology congress.
+
+## Built With
+
+- Major languages - HTML, CSS and JavaScript
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://ashe546.github.io/capstone-1/)
```

Comment:
Nice demo! :+1:  However, it's still missing a video that shows your project. 

- [ ] A video with a maximum length of 5 minutes. The content of the video should include:

 - A description of the project.
 - A demo of the project features (different pages and different screen sizes).
 - You should also highlight some interesting piece of code or something you built that you are very proud of.

This requirement is a big opportunity to practice your communication skills. Don't be shy!